images/tkg/README.md

```diff
@@ -92,14 +92,14 @@ make clean IMAGE_ARTIFACTS_PATH=/root/artifacts/ LABEL=byoi_image_builder # To c
 
 - `make run-artifacts-container` is used to run the artifacts container for a Kubernetes version at a particular port
```

Comment:
Should we mention here in one liner on what artifacts container contains?

Reply:
I will try to add a separate document on how things work in a separate PR (Like an architecture document) that will have more information on what the container contains etc.